package.json

```diff
@@ -59,7 +59,6 @@
     "dependencies": {
         "@inertiajs/inertia": "^0.11.0",
         "@inertiajs/inertia-vue3": "^0.6.0",
-        "@vue/compat": "^3.3.8",
         "@wmde/wikit-tokens": "^2.1.0-alpha.15",
         "@wmde/wikit-vue-components": "^2.1.0-alpha.16",
```

Comment:
I think we can remove lines 62 and 63 too

Reply:
I think removing `wikit` is not in the scope of this ticket, that's why I left these lines.